tests/orgsGetByTag.test.js

```diff
@@ -64,3 +144,52 @@ test('Get all the active organisations, associated with a specific tag', functio
     });
   });
 });
+
+test('When no orgs are found to match a specific tag', function (t) {
+  var tagId = 2;
+  init(config, function (err, server, pool) {
+    if (err) {
+      console.log('error initialise server', err);
+      return t.fail();
+    }
+
+    server.inject({
+      method: 'GET',
+      url: '/orgsGetByTag?tags=' + tagId
+    }, function (res) {
+      var expected = {
+        filter_tag: 'Corporate',
+        orgs: [{
+          id: null,
+          name: null,
+          logo_url: null,
+          active: null
+        }]
```

Comment:
@mk4111 i like this test, but it seems to make most sense that in this case, that orgs is an empty array. That way orgs.length will always equal the number of orgs returned.